src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,1864 @@
+                            });
+                    });
+                } else if (value === 'MetaESDT') {
+                    const address: string = 'erd1rf4hv70arudgzus0ymnnsnc4pml0jkywg2xjvzslg0mz4nn2tg7q7k0t6p';
+                    it(`should return alternative count of MetaESDT tokens for a given address ${address}, when filter includeMetaESDT is true`, async () => {
+                        const params = new URLSearchParams({
+                            'includeMetaESDT': 'true',
+                        });
+
+                        await request(app.getHttpServer())
+                            .get(`${path}/${address}/tokens/c?${params}&${filter}=${value}`)
+                            .expect(200)
+                            .then(res => {
+                                expect(+res.text).toBeGreaterThanOrEqual(1);
+                            });
+                    });
+                    it(`should return alternative count of MetaESDT tokens for a given address ${address}, when filter includeMetaESDT is false`, async () => {
+                        const params = new URLSearchParams({
+                            'includeMetaESDT': 'false',
+                        });
+
+                        await request(app.getHttpServer())
+                            .get(`${path}/${address}/tokens/c?${params}&${filter}=${value}`)
+                            .expect(200)
+                            .then(res => {
+                                expect(+res.text).toStrictEqual(0);
+                            });
+                    });
+                }
+            });
+        });
+    });
+
+    describe('/accounts/{address}/tokens/{token}', () => {
+        it('should return 400 Bad Request for an invalid token', async () => {
+            const address: string = 'erd1rf4hv70arudgzus0ymnnsnc4pml0jkywg2xjvzslg0mz4nn2tg7q7k0t6p';
+            const token: string = 'AER';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}/tokens/${token}`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toStrictEqual("Validation failed for argument 'token': Invalid token / NFT identifier.");
+                });
+        });
+
+        it('should return 400 Bad Request for an invalid address', async () => {
+            const address: string = 'erd1rf4hvymnnsnc4pml0jkywg2xjvzslg0mz4nn2tg7q7k0t6p';
+            const token: string = 'AERO-458bbf';
+
+            await request(app.getHttpServer())
+                .get(`${path}/${address}/tokens/${token}`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toStrictEqual("Validation failed for argument 'address' (a bech32 address is expected)");
+                });
+        });
+
+        it('should return 404 Not Found for a non-fungible token from a given address', async () => {
```

Comment:
here should be MetaESDT
Resource: https://explorer.multiversx.com/nfts/LKMEX-aab910-323141